third_party/blink/renderer/core/html/media/html_media_element.cc

```diff
@@ -405,26 +405,51 @@ std::ostream& operator<<(std::ostream& stream,
   return stream << static_cast<void const*>(&media_element);
 }
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+// Checks for progressive formats served by the YouTube H5 player.
+// These formats have a mime type of "video/mp4", and lists both audio and
+// video formats under the "codecs" parameter.
+bool IsProgressiveFormat(const ContentType& content_type) {
+  String type = content_type.GetType();
+  String codecs = content_type.Parameter("codecs");
+  if (!type.empty() && !codecs.empty()) {
```

Comment:
micro-nit: Prefer early return for readability, i.e.:
```
if (type.empty() || codecs.empty()) {
  return false;
}
```

Reply:
Done, and I made them const